utils/database.py

```diff
@@ -18,23 +22,33 @@ def _init_(self):
         self.engine = create_engine(os.getenv("DATABASE_URL"), echo=bool(os.getenv("DATABASE_LOG")))
         Base.metadata.create_all(self.engine)
 
-    # Function is only for showing how orm work
-    # TODO: remove later
-    def add_log(self, server_guid: int, text: str):
+    def add_time_on_channel(self, server_id: int, channel_id: int, user_id: int, timeChannel: int):
```

Comment:
I'm afraid that if we will ad all data base operations in here, this file will be giaiant. 


To do: split database.py into many modules base on functionality (eg. Voice statisic)